docs/ga-roadmap.md

```diff
@@ -21,7 +21,7 @@ All stabilized modules will conform to the API expectations outlined in the [VER
 Explicitly, the following are not in the scope of v1 for the purposes of this document:
 
 * stabilization of additional components/APIs needed by distribution maintainers. Vendors are not the audience
-  * This explicitly excludes the `service` and `otelcol` modules.
+  * This explicitly excludes the `service` and `otelcol` modules, for which we only aim to guarantee behavior stability.
```

Comment:
Is there a reference that explains "behavior stability" further?

As I understand, a change in `service` config is considered a behavior change, while a change in Go API is not a behavior change, but somewhere in between things can become vague. E.g. what if a Go API change in `service` leads to change in how fatal error is reported etc.

Reply:
What about the following phrasing?
> We will only guarantee that there are no breaking changes impacting end-users of the binary after 1.0, while Go API only changes will continue to be admissible until the modules are tagged as 1.0

Does this make more sense to you?

Reply:
Thx, that makes more sense 👍